faunadb-java/src/test/java/com/faunadb/client/JavaClientSpec.java

```diff
@@ -14,152 +15,67 @@
 import com.google.common.base.Optional;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.util.concurrent.ListenableFuture;
 import org.joda.time.Duration;
 import org.joda.time.Instant;
 import org.joda.time.LocalDate;
+import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 
 import java.util.Arrays;
-import java.util.Random;
+import java.util.List;
 
+import static com.faunadb.client.database.SetupDatabase.*;
 import static com.faunadb.client.query.Language.Action.CREATE;
 import static com.faunadb.client.query.Language.Action.DELETE;
 import static com.faunadb.client.query.Language.*;
+import static com.faunadb.client.query.Language.Class;
 import static com.faunadb.client.query.Language.TimeUnit.*;
 import static com.faunadb.client.types.Codec.*;
+import static com.faunadb.client.types.Value.NullV.NULL;
+import static com.google.common.base.Functions.constant;
+import static com.google.common.util.concurrent.Futures.catching;
+import static java.lang.String.format;
 import static org.hamcrest.CoreMatchers.*;
 import static org.hamcrest.collection.IsArrayContainingInOrder.arrayContaining;
 import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
 import static org.hamcrest.collection.IsIterableContainingInOrder.contains;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.Is.isA;
 import static org.joda.time.DateTimeZone.UTC;
 import static org.junit.Assert.assertThat;
 
-public class ClientSpec extends FaunaDBTest {
+public class JavaClientSpec extends DslSpec {
+  private static final String ROOT_TOKEN = EnvVariables.require("FAUNA_ROOT_KEY");
+  private static final String ROOT_URL = format("%s://%s:%s",
+    EnvVariables.getOrElse("FAUNA_SCHEME", "https"),
+    EnvVariables.getOrElse("FAUNA_DOMAIN", "cloud.faunadb.com"),
```

Comment:
Old domain here.

Should we maybe share these test constant  statics somehow between modules? I don't think its a huge deal.